about.html

```diff
@@ -1,73 +1,136 @@
 <!doctype html>
 
 <html lang="en">
+
 <head>
-  <meta charset="utf-8">
+    <meta charset="utf-8">
 
-  <title>Sprint Challenge - About</title>
+    <title>Sprint Challenge - About</title>
 
-  <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
-  <link rel="stylesheet" href="css/index.css">
+    <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
+    <link rel="stylesheet" href="css/about.css">
 
 </head>
 
 <body>
     <div class="container about-page">
-     
-    About
 
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
+        <nav class="navigation">
+            <a href="#">Home</a>
+            <a href="about.html">About</a>
+            <a href="#">Products</a>
+            <a href="#">Blog</a>
+            <a href="#">Contact</a>
 
+            <a href="index.html"><img class="logo" src="img/lambda-black.png" alt="lambda-logo"></a>
 
-    <img src="img/about-plan.png" alt="strategy">
-    
-    Strategy
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
+        </nav>
 
-    Learn More
+        <img class="jumbo-about" src="img/jumbo-about.png" alt="jumbo-about"></img>
 
+        <div class="text-container">
+            <h2>About</h2>
+            <p>Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of
+                disruptive innovation via workplace diversity and empowerment.</p>
 
+        </div>
+        <hr>
 
-    <img src="img/about-working.png" alt="strategy">
-    
-    How We Work
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
+        <section class="about-top">
 
-    Learn More
+            <div class="strategy-top">
+                <img class="plan" src="img/about-plan.png" alt="strategy"></img>
 
+                <section class="strategy">
```

Comment:
You actually don't need to wrap all of your elements in section tags - section tags are used for full sections of content. It would be better to get rid of the section tag, and to instead put the "strategy" class on the h3.

Reply:
Couldn't fix this